docs/src/main/sphinx/installation/jdbc.rst

```diff
@@ -167,4 +167,5 @@ Name                                                         Description
 ``externalAuthentication``                                   Use a local web browser to authenticate with an identity provider (IdP)
                                                              that has been configured for the Trino coordinator.
                                                              See :doc:`/security/oauth2` for more details.
+``disableCompression``                                       Whether compression should be enabled.
```

Comment:
What kind of compression? Why would anyone disable it?

Reply:
@sopel39 see https://github.com/trinodb/trino/pull/5818